src/utility/sdk.js

```diff
@@ -32,12 +32,27 @@ const parseServiceMetadata = (response) => {
 };
 
 const fetchServiceMetadata = async (org_id, service_id) => {
+  if(process.env.REACT_APP_SANDBOX) {
```

Comment:
process.env variables would always be casted to Strings.. so this `might` be always true.

Reply:
It will only be casted if the value is provided. But since this is a flag that we are setting in `package.json`, it will only be set for sandbox command. This works fine.